mct_quantizers/common/constants.py

```diff
@@ -72,11 +72,13 @@
 
 
 ## Constant values
+
 LAYER = "layer"
 STEPS = "optimizer_step"
 TRAINING = "training"
 EPS = 1e-8
 LUT_VALUES_BITWIDTH = 8
+MCTQ_VERSION = "mctq_version"
 
```

Comment:
Can you please use this constant in pytorch as well?
https://github.com/sony/mct_quantizers/blob/main/mct_quantizers/pytorch/quantizers/base_quantizer_autograd_function.py#L57